osdconfig/osdutil/io.go

```diff
@@ -0,0 +1,168 @@
+// package osdutil is a higher level abstraction on osdconfig.
+// It allows easier interface to setup kvdb watches
+package osdutil
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/pkg/errors"
+	"github.com/portworx/kvdb"
+	"github.com/sdeoras/openstorage/osdconfig"
```

Comment:
import should be from libopenstorage. Same in other places in the change.

Reply:
yes, I realized it bit too late yesterday when CI was failing. Since `osdconfig` is not yet pushed to master, I only had it in the forked repo and goimports picked it up.